speech/src/main/java/com/example/speech/InfiniteStreamRecognize.java

```diff
@@ -39,19 +40,16 @@
 import javax.sound.sampled.AudioSystem;
 import javax.sound.sampled.DataLine;
 import javax.sound.sampled.DataLine.Info;
+import javax.sound.sampled.LineUnavailableException;
 import javax.sound.sampled.TargetDataLine;
 
 public class InfiniteStreamRecognize {
 
   private static final int STREAMING_LIMIT = 290000; // ~5 minutes
-
-  public static final String RED = "\033[0;31m";
-  public static final String GREEN = "\033[0;32m";
-  public static final String YELLOW = "\033[0;33m";
+  private static final String EXIT_WORD = "exit";
 
   // Creating shared object
   private static volatile BlockingQueue<byte[]> sharedQueue = new LinkedBlockingQueue<byte[]>();
-  private static TargetDataLine targetDataLine;
   private static int BYTES_PER_BUFFER = 6400; // buffer size in bytes
```

Comment:
nit: Consider selecting multiple of 2 for the buffer size as a general guideline. Is it possible to explain how to select a certain value?

Reply:
I can't explain how to select a certain value.